doc.md

```diff
@@ -0,0 +1,110 @@
+# Hackseq 2017
+
+The system allows users to run graphical user interface application (e.g. `igv`), provides 100 GB hard memory and 20 GB soft memory, maps host users to Docker users inside containers and mounts team folders. 
+
+## Overview
+
+**Servers**: orca01.dmz.bcgsc.ca, orca02.dmz.bcgsc.ca
+* 5 teams per a server
+* a team shares a team’s folder
+* the Docker container per a team
+* each user has a home directory
+* ability to run GUI applications (X11)
+* the groups’ user name is the same as a name of the team
+
+## Group creation
+
+Add groups: `orca_users:*:1000:`, name of teams (e.g. `hackseq17_1:*:1002:`)
+
+## User creation
+* Primary group name = a name of the team = a name of the Docker container
+* Secondary group name = `1000(orca_users)`
+* User home directory: 
+    1) create empty `.Xauthority file` (X11)  
+    2) to make `brew install` work:
+    ```
+    mkdir -p -m775 ~/.cache/Homebrew 
+    ```
+    and copy  `.bashrc` from `/orca_repo/etc`
+* Make sure that in `.bashrc` there is the following line:
+```
+alias brew='sudo -u linuxbrew HOMEBREW_NO_AUTO_UPDATE=1 /home/linuxbrew/.linuxbrew/bin/brew'
+```
+* Add new entries from outputs of `getent passwd` and `getent group` commands to `group_hs17` and `passwd_hs17` files in `/orca_repo/etc`.
+
+* Add `hosts` file to the `/orca_repo/etc`:
+```
+127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
+::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
+10.9.235.200 orca ldap.orca.ca biodocker01 biodocker01.dmz.bcgsc.ca
+10.9.235.202 orca02 orca02.dmz.bcgsc.ca
+```
+## Containers creation
+
+Use the following script to create a container for each team. Set `PROJECT` variable for each team and run the Docker command:
+```
+FILES_LOCATION=/orca_repo/etc
+PROJECT=hackseq17_6
+
+docker run -t --name $PROJECT -m 100GB --memory-reservation 20GB -v $FILES_LOCATION/hosts:/etc/hosts:ro -v /projects/hackathon/2017:/home/projects -v $FILES_LOCATION/passwd_hs17:/etc/passwd:ro -v $FILES_LOCATION/group_hs17:/etc/group:ro -w /home/projects/$PROJECT -v /home/userhome:/home/userhome -v /tmp/.X11-unix:/tmp/.X11-unix --net host bcgsc/orca-hackseq:2017.10.12
+```
```

Comment:
Please put this code in a shell script named `hackseq-create-group`, commit it to GitHub, and link to it from this documentation.

Reply:
but it is not so convenient to jump from one link to another. In that case we don't see whole picture. There is way to show a shell script of the link ?

Reply:
Not a simple way using just Markdown. Summarize the purpose of the script and its implementation. The reader can click on the link to get the details.